examples/demo-app/src/factories/load-data-modal.js

```diff
@@ -29,6 +29,7 @@ const CustomLoadDataModalFactory = (...deps) => {
   // add more loading methods
   const loadingMethods = [
     defaultLoadingMethods.find(lm => lm.id === 'upload'),
+    defaultLoadingMethods.find(lm => lm.id === 'tileset'),
```

Comment:
Nit: Feels more like a `category` than an `id`...

```suggestion
    defaultLoadingMethods.find(method => method.id === 'tileset'),
```